app/uk/gov/hmrc/hmrcemailrenderer/templates/tcs/newTcsRenewalConfirmationMessageAlert.scala.html

```diff
@@ -1,24 +1,106 @@
+        </ul>
+    } else {
+        @if(params("incompleteClaims") == "1") {
+            <p style="font-size: 16px;line-height: 1.315789474;">You must:</p>
+            <ul>
+                <li style="font-size: 16px;line-height: 1.315789474; margin: 0 0 10px 0;">complete your @{params("incompleteClaims")} remaining annual review to finish your renewal.</li>
+            </ul>
+        }
+    }
+}
 
-<p style="font-size: 19px;line-height: 1.315789474; margin: 0 0 30px 0;">Do not send us your renewal pack through the post, as this will delay your renewal.</p>
+@if(params("isMultiple") == "false"){
+    @if((params("anyEstimates") != "0") && (params("anyEstimates") != "1")){
+        <p style="font-size: 16px;line-height: 1.315789474;">You must:</p>
+        <ul>
+            <li style="font-size: 16px;line-height: 1.315789474; margin: 0 0 10px 0;">contact the tax credits helpline to tell us the actual amount for the estimates you gave by 31 January 2017.</li>
+        </ul>
+    } else {
+        @if(params("anyEstimates") == "1") {
+            <p style="font-size: 16px;line-height: 1.315789474;">You must:</p>
+            <ul>
+                <li style="font-size: 16px;line-height: 1.315789474; margin: 0 0 10px 0;">contact the tax credits helpline to tell us the actual amount for the estimate you gave by 31 January 2017.</li>
+            </ul>
+        }
+    }
+}
+
+<p style="font-size: 16px;line-height: 1.315789474;">You can:</p>
+<ul>
+    <li style="font-size: 16px;line-height: 1.315789474; margin: 0 0 10px 0;">check the status of your renewal</li>
+    <li style="font-size: 16px;line-height: 1.315789474; margin: 0 0 10px 0;">complete this review again if you think you made a mistake</li>
+</ul>
 
 <h2>What happens next</h2>
 
-<p style="font-size: 19px;line-height: 1.315789474;">We will work out how much tax credits you:</p>
+<p style="font-size: 16px;line-height: 1.315789474;">We will work out how much tax credits you:</p>
 
 <ul>
-    <li style="font-size: 19px;line-height: 1.315789474; margin: 0 0 10px 0;">should have got</li>
-    <li style="font-size: 19px;line-height: 1.315789474; margin: 0 0 10px 0;">will get from @{params("startDate")} to @{params("endDate")}, unless your claim has ended</li>
+    <li style="font-size: 16px;line-height: 1.315789474; margin: 0 0 10px 0;">should have got in your award period</li>
+    <li style="font-size: 16px;line-height: 1.315789474; margin: 0 0 10px 0;">will get from @{params("startDate")} to @{params("endDate")}, unless your claim has ended,
+        and send you an award notice that you should get by @{params("noticeDate")}</li>
 </ul>
 
-<p style="font-size: 19px;line-height: 1.315789474; margin: 0 0 30px 0;">We will send you an award notice that you should get by @{params("noticeDate")}.</p>
+<p style="font-size: 16px;line-height: 1.315789474; margin: 0 0 30px 0;">If the amount was wrong, we will pay you the right amount or ask you to pay back any extra money.</p>
 
```

Comment:
As per contributing guidelines (point 5 -> https://github.com/hmrc/hmrc-email-renderer/blob/master/CONTRIBUTING.md#content-recommendations) here you should inform the customer why he's receiving this email